include/Primitives/Plane.hpp

```diff
@@ -0,0 +1,54 @@
+/*
+** EPITECH PROJECT, 2024
+** Raytracer
+** File description:
+** Plane
+*/
+
+#pragma once
+
+#include "IPrimitive.hpp"
+
+namespace Primitive {
+
+    enum Axis {
+        X,
+        Y,
+        Z
+    };
+
+    class Plane : public IPrimitive {
+
+        public:
+
+            /**
+             * @brief Construct a new Plane object
+             *
+             */
+            Plane();
+
+            /**
+             * @brief Construct a new Plane object
+             *
+             */
+            Plane(Primitive::Axis axis, double position);
```

Comment:
Hello Mister Tranchet,

Can you please add comment about the variable that you use in your documentation